packages/eslint-config-skuba/package.json

```diff
@@ -34,10 +34,13 @@
     "typescript": "~5.3.0"
   },
   "peerDependencies": {
-    "eslint": ">= 7.2",
-    "typescript": ">= 3.9"
+    "eslint": ">=8.56.0",
+    "typescript": ">=4.7.5"
```

Comment:
Should this be 4.7.4 or 4.7.5?

Reply:
ruh oh, should be 4.7.4 but I guess I mucked that up in eslint-config-seek

Reply:
Ah no stress, we can align on 4.7.5 👍